x/marker/types/msg.go

```diff
@@ -515,6 +523,54 @@ func (msg MsgTransferRequest) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{adminAddr}
 }
 
+// NewIbcMsgTransferRequest
+func NewIbcMsgTransferRequest(
+	administrator string,
+	sourcePort, sourceChannel string,
+	token sdk.Coin, sender, receiver string,
+	timeoutHeight clienttypes.Height, timeoutTimestamp uint64, //nolint:interfacer
+) *MsgIbcTransferRequest {
+	return &MsgIbcTransferRequest{
+		Administrator: administrator,
+		Transfer: ibctransfertypes.MsgTransfer{
+			SourcePort:       sourcePort,
+			SourceChannel:    sourceChannel,
+			Token:            token,
+			Sender:           sender,
+			Receiver:         receiver,
+			TimeoutHeight:    timeoutHeight,
+			TimeoutTimestamp: timeoutTimestamp,
+		},
+	}
+}
+
+// Route returns the name of the module.
+func (msg MsgIbcTransferRequest) Route() string { return ModuleName }
+
+// ValidateBasic runs stateless validation checks on the message.
+func (msg MsgIbcTransferRequest) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Administrator); err != nil {
+		return err
+	}
+	return msg.Transfer.ValidateBasic()
+}
+
+// GetSignBytes encodes the message for signing.
+func (msg MsgIbcTransferRequest) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(&msg)
+	return sdk.MustSortJSON(bz)
+}
+
+// GetSigners indicates that the message must have been signed by the address provided.
+func (msg MsgIbcTransferRequest) GetSigners() []sdk.AccAddress {
+	adminAddr, err := sdk.AccAddressFromBech32(msg.Administrator)
+	if err != nil {
+		panic(err)
```

Comment:
nit: if we use `MustAccAddressFromBech32`  we can skip line 567 and 568 i think

Reply:
True, I will change the others in this file to match.